view/src/main/java/cmm/apps/esmorga/view/eventlist/mapper/EventUiMapper.kt

```diff
@@ -0,0 +1,38 @@
+package cmm.apps.esmorga.view.eventlist.mapper
+
+import cmm.apps.esmorga.domain.event.model.Event
+import cmm.apps.esmorga.view.eventdetails.model.EventDetailsUiState
+import cmm.apps.esmorga.view.eventlist.model.EventListUiModel
+import java.time.ZonedDateTime
+import java.time.format.DateTimeFormatter
+import java.util.TimeZone
+
+object EventUiMapper {
+
+    private fun formatDate(date: ZonedDateTime) : String{
+        return date.format(DateTimeFormatter.ofPattern("d' de 'MMMM' a las 'HH:mm").withZone(TimeZone.getDefault().toZoneId()))
```

Comment:
Would be nice to be able to move this string to strings.xml in some way. Maybe with ordered parameters or similar.